__tests__/api.test.js

```diff
@@ -282,3 +283,25 @@ describe("6. GET /api/articles", () => {
       });
   });
 });
+
+describe("12. DELETE /api/comments/:comment_id", () => {
+  test("should return status 204 ", () => {
+    return request(app).delete("/api/comments/1").expect(204);
```

Comment:
Would be nice to check at this stage that the comment has actually been deleted - either by making a get request to the endpoint for that comment or trying to delete it again - and checking that you'll get back what you're expecting.